django/db/models/sql/subqueries.py

```diff
@@ -215,7 +215,5 @@ class AggregateQuery(Query):
     compiler = 'SQLAggregateCompiler'
 
     def add_subquery(self, query, using):
-        self.subquery, self.sub_params = query.get_compiler(using).as_sql(
-            with_col_aliases=True,
-            subquery=True,
-        )
+        query.subquery = True
```

Comment:
I'm not totally sure if this is safe - is it possible the same query is used elsewhere where it shouldn't be a subquery.

Maybe we could pass the subquery as as_sql() parameter?